lib/slack/sends.ex

```diff
@@ -68,6 +68,18 @@ defmodule Slack.Sends do
       |> send_raw(slack)
   end
 
+  @doc """
+  Subscribe to presence notifications for the user IDs in `ids`.
+  """
+  def subscribe_presence(ids \\ [], slack) do
+    %{
+      type: "presence_sub",
+      ids: ids
+    }
+      |> Poison.encode!()
```

Comment:
This indentation looks off, can you align it with `Poison.encode!`?

Reply:
Are you sure? It's currently matching all the other similar functions in that file.

see `send_ping`, `indicate_typing`, `send_message`

Reply:
I propose to merge as is, then run the 1.6 code formatter later to deal with style discrepancies.

Reply:
Agreed, no longer worth bike shedding over it with 1.6 having been released. 👍